guide/ops/configuration/brooklyn_cfg.md

```diff
@@ -33,7 +28,7 @@ brooklyn.webconsole.security.user.admin.password=AdminPassw0rd
 brooklyn.webconsole.security.user.bob.password=BobPassw0rd
 {% endhighlight %}
 
-The properties file *must* have permissions 600 
+The config file *must* have permissions 600 
```

Comment:
I don't think that is true anymore. Although, as long as it is readable and writable for the file owner, it should be fine